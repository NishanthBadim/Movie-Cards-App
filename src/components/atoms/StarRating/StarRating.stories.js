import React from 'react'
import { withKnobs, number } from '@storybook/addon-knobs/react'
import StarRating from './StarRating'

export default {
    component: StarRating,
    title: "StarRating",
    decorators: [withKnobs]
}

export const Default = () => {
    return (
        <StarRating rating={number("rating", 5.0)} />
    )
}