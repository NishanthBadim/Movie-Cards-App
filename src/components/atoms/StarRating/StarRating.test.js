import React from 'react';
import {create} from 'react-test-renderer'
import "@testing-library/jest-dom/extend-expect";
import StarRating from './StarRating'

test("Match the snapshot - StarRating",() => {
    const input=create(<StarRating />);
    expect(input.toJSON()).toMatchSnapshot();
})