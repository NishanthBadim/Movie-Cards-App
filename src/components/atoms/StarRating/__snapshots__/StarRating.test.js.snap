// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match the snapshot - StarRating 1`] = `
<div>
  <div
    style={
      Object {
        "overflow": "hidden",
      }
    }
  >
    <div
      style={
        Object {
          "width": "0px",
        }
      }
    >
      <div
        style={
          Object {
            "position": "absolute",
            "width": "110px",
          }
        }
      >
        <i
          className="fa fa-star-o fa-lg"
          style={
            Object {
              "padding": "1px",
            }
          }
        />
        <i
          className="fa fa-star-o fa-lg"
          style={
            Object {
              "padding": "1px",
            }
          }
        />
        <i
          className="fa fa-star-o fa-lg"
          style={
            Object {
              "padding": "1px",
            }
          }
        />
        <i
          className="fa fa-star-o fa-lg"
          style={
            Object {
              "padding": "1px",
            }
          }
        />
        <i
          className="fa fa-star-o fa-lg"
          style={
            Object {
              "padding": "1px",
            }
          }
        />
      </div>
      <div
        style={
          Object {
            "width": "110px",
          }
        }
      >
        <i
          className="fa fa-star fa-lg"
          style={
            Object {
              "padding": "1px",
            }
          }
        />
        <i
          className="fa fa-star fa-lg"
          style={
            Object {
              "padding": "1px",
            }
          }
        />
        <i
          className="fa fa-star fa-lg"
          style={
            Object {
              "padding": "1px",
            }
          }
        />
        <i
          className="fa fa-star fa-lg"
          style={
            Object {
              "padding": "1px",
            }
          }
        />
        <i
          className="fa fa-star fa-lg"
          style={
            Object {
              "padding": "1px",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;
